### PR TITLE
Client inventory implemented

### PR DESCRIPTION
**Pull request by ArushiSharma95**
FOCUS: 272- Client Inventory implementation

### CODE REVIEW

File: frontend/src/features/clients/ClientList.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import { useGetConsultantClientsQuery } from "features/clients/clientSlice";
+import { ClientInventory } from "components";
+import { store } from "../../app/store";
```
Comment (jsnkle) [inline frontend/src/features/clients/ClientList.js:3]: I'll give you three guesses what I'm gonna call out here. lol

File: frontend/src/features/clients/clientSlice.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { apiSlice } from "app/api";
+
+const clientSlice = apiSlice.injectEndpoints({
+	endpoints: (build) => ({
+		getConsultantClients: build.query({
+			query: (consultantId) => `accounts/consultantclients/${consultantId}`,
```
Comment (jsnkle) [inline frontend/src/features/clients/clientSlice.js:6]: We probably don't need a providesTags here for the MVP but at some point we may need to add that in.

File: frontend/src/features/clients/ClientList.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+	});
+
+	if (isSuccess) {
+		clientLists = data.data;
+	}
+	return (
+		<>
+			<div className='row'>
+				{clientLists.map((clientList, i) => {
+					if (i < 4 && clientList) {
```
Comment (jsnkle) [inline frontend/src/features/clients/ClientList.js:22]: This is okay for the MVP but we may want to talk about how we will handle this if for some reason a given consultant has more than 4 active clients assigned to them.

**Review by jsnkle — CHANGES_REQUESTED** [review]

File: frontend/src/features/clients/ClientList.js (hunk 3 above)
Comment (jsnkle) [reply frontend/src/features/clients/ClientList.js:22]: Also... this should be i <= 4

File: frontend/src/features/clients/clientSlice.js (hunk 2 above)
Comment (ArushiSharma95) [reply frontend/src/features/clients/clientSlice.js:6]: That is right Jason. We can add it later when needed.

File: frontend/src/features/clients/ClientList.js (hunk 3 above)
Comment (ArushiSharma95) [reply frontend/src/features/clients/ClientList.js:22]: This starts with 0.

**Review by jsnkle — APPROVED** [review]

**Pull request merged**